symbols/ram.yml

```diff
@@ -223,6 +223,15 @@ arm9:
         A null-terminated string, with a maximum length of 10. Presumably encoded with the ANSI/Shift JIS encoding the game typically uses.
         
         This is presumably part of a larger struct, together with other nearby data.
+    - name: SCRIPT_VARS_VALUES
```

Comment:
[The formatter](https://github.com/UsernameFodder/pmdsky-debug/runs/5266440927?check_suite_focus=true) caught this as being out of order.

Reply:
The one time I *don't* run it...

Reply:
Oh wait, that's the one I added before, it didn't catch it, I'll check your link and see if I did it wrong.